examples/scf_properties/atomic_charges/httpx/run.py

```diff
@@ -0,0 +1,138 @@
+import base64
+import json
+import httpx
+import os
+
+from promethium_sdk.utils import wait_for_workflows_to_complete
+
+foldername = "output"
+base_url = os.getenv("PM_API_BASE_URL", "https://api.promethium.qcware.com")
+gpu_type = os.getenv("PM_GPU_TYPE", "a100")
+
+if not os.path.exists(foldername):
+    os.makedirs(foldername)
+
+headers = {
+    "x-api-key": os.environ["PM_API_KEY"],
+    "accept": "application/json",
+    "content-type": "application/json",
+}
+
+client = httpx.Client(base_url=base_url, headers=headers)
+
+input_mol = base64.b64encode(
+    b"""
```

Comment:
Let's make this a valid XYZ string i.e., include the atom count. Technically this is not valid, as it doesn't conform to the header spec of line 1 -- atom count, line 2 -- comment. We do accept it, but all examples should encourage correct specs